src/Carbon/Carbon.php

```diff
@@ -112,6 +112,11 @@ class Carbon extends DateTime
     const SECONDS_PER_MINUTE = 60;
 
     /**
+     * RFC7231 DateTime format.
+     */
+    const RFC7231 = 'D, d M Y H:i:s \G\M\T';
```

Comment:
IMHO @neclimdul's variant is more appropriate in this case. Because when you just look at the const value - you see there "timezone identifyer" whilst RFC says us that there should be "exactly GMT".